### PR TITLE
adding version aliases for release versions and git remote option

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: deploy.rb [hunk 1]
```diff
@@ -47,16 +51,26 @@ def make_deploymentdir
...
+  def clone_versions(remote)
+    @versions.each { |version| clone_version(version, remote) }
   end
 
-  def clone_version(version)
+  def clone_version(version, remote)
+    if remote == 'origin'
+      remote_url = 'git://github.com/Katello/katello.org' 
+    else   
+      remote_url = syscall("git remote  show jlsherri | grep Fetch | awk '{print $3}'")
```
Comment (stbenjam) [inline deploy.rb]: How about `git config --get remote.#{remote}.url` ?

Comment (stbenjam) [inline deploy.rb]: Or maybe `git ls-remote --get-url #{remote}` is better, but anything with out jlsherri :smile:

Comment (jlsherrill) [inline deploy.rb]: hah, oops!

**Comment by stbenjam** [comment]
ACK

**Pull request merged**